test/parallel/test-domain.js

```diff
@@ -259,7 +259,7 @@ const fst = fs.createReadStream('stream for nonexistent file');
 d.add(fst);
 expectCaught++;
 
-[42, null, , false, common.noop, 'string'].forEach(function(something) {
```

Comment:
Maybe it was meant to add `undefined`?
Also re: #12822 `/s/common.noop/() => {}/`